vllm/config/__init__.py

```diff
@@ -471,7 +471,7 @@ class ModelConfig:
     """One or more logits processors' fully-qualified class names or class
     definitions"""
 
-    def compute_hash(self) -> str:
+    def compute_hash(self, extra_ignored_fields: Iterable[str] = ()) -> str:
```

Comment:
2 things:
- Why not make this a set?
- Mutable defaults are dangerous

```suggestion
    def compute_hash(self, extra_ignored_fields: set[str] = None) -> str:
```